plugin-server/src/main/ingestion-queues/analytics-events-ingestion-consumer.ts

```diff
@@ -91,6 +98,11 @@ export async function eachBatchIngestionWithOverflow(
                 // Set message key to be null so we know to send it to overflow topic.
                 // We don't want to do it here to preserve the kafka offset handling
                 message.key = null
+
+                if (LoggingLimiter.consume(seenKey, 1) === true) {
+                    ingestionPartitionKeyOverflowed.labels(seenKey).inc()
```

Comment:
Can we put this outside of the if? We aren't concerned about limiting the amount we record in metrics I don't think.

Reply:
Of course 👍